components/brave_wallet_ui/constants/types.ts

```diff
@@ -139,3 +139,8 @@ export interface WalletPanelState {
   wallet: WalletState
   panel: PanelState
 }
+
+export interface RecoveryObjectType {
```

Comment:
nit: I think we should cease appending `Type` to every typescript definition. First, this isn't a `type`, it's an `interface`. Plus, it's implied when `interfaces` and `types` can only be used in certain syntactic places in typescript. Also, other typescript types, like enums translate to actual JS objects, so it's not strictly always a "Type".

Reply:
Updated!